doc/default.nix

```diff
@@ -1,6 +1,6 @@
 { pkgs ? (import ./.. { }), nixpkgs ? { }}:
 let
-  doc-support = import ./doc-support { inherit pkgs nixpkgs; };
+  doc-support = import ./doc-support { inherit pkgs; nixpkgs-revision = nixpkgs.revision; };
```

Comment:
Does this work with `[nixpkgs]$ nix-build ./doc`?

Reply:
No, it doesn't.

I guess there's some ambiguity about what are valid entry points into nixpkgs.  Maybe this should wait until that question gets answered.